core/src/main/scala/cats/Composed.scala

```diff
@@ -1,6 +1,14 @@
 package cats
 
 
+private [cats] trait ComposedDistributive[F[_], G[_]] extends Distributive[λ[α => F[G[α]]]] with ComposedFunctor[F, G] { outer =>
```

Comment:
Scalastyle doesn't like the space before `[cats]` here.

Reply:
fixed.